internal/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -95,6 +97,9 @@ RUN TINIARCH=$(case $(arch) in x86_64) echo -n amd64 ;; ppc64le) echo -n ppc64el
   && curl -L -o /tini https://github.com/krallin/tini/releases/latest/download/tini-$TINIARCH \
   && chmod +x /tini
 
+[[- if .Requirements ]]
+COPY requirements.yml ${HOME}/requirements.yml
+RUN ansible-galaxy collection install -r ${HOME}/requirements.yml[[ end ]]
```

Comment:
We need to add it in the migration guide. See: https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#unreleased

Reply:
:+1: done